lib/package.rb

```diff
@@ -245,23 +253,15 @@ def self.get_sha256(architecture)
   end
 
   def self.get_extract_dir
-    "#{name}.#{Time.now.utc.strftime('%Y%m%d%H%M%S')}.dir"
+    return "#{name}.#{Time.now.utc.strftime('%Y%m%d%H%M%S')}.dir"
   end
 
   def self.is_binary?(architecture)
-    if !@build_from_source && @binary_url && @binary_url.key?(architecture)
-      return true
-    else
-      return false
-    end
+    return (!@build_from_source && @binary_url && @binary_url.key?(architecture))
   end
 
   def self.is_source?(architecture)
-    if is_binary?(architecture) || is_fake?
-      return false
-    else
-      return true
-    end
+    return !(is_binary?(architecture) || is_fake?)
```

Comment:
If these are just general improvements to the code,
does it make more sense to seperate these into another PR?